docs/source/openvino/export.mdx

```diff
@@ -109,6 +96,35 @@ Models larger than 1 billion parameters are exported to the OpenVINO format with
 
 </Tip>
 
+
+### Decoder models
+
+```bash
+optimum-cli export openvino --model  meta-llama/Meta-Llama-3-8B --task text-generation-with-past ov_llama/
+```
+
+The `-with-past` suffix enable the re-use of past keys and values. This allows to avoid recomputing the same intermediate activations during the generation. to export the model without, you will need to remove this suffix.
```

Comment:
```suggestion
The `-with-past` suffix enable the re-use of past keys and values. This allows to avoid recomputing the same intermediate activations at each generation step. To export the model without, you will need to remove this suffix.
```